CHANGELOG.md

```diff
@@ -21,7 +21,10 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-3.1.4...master
+[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-3.1.5...master
+
+## 3.1.5 - 2024-06-12
+It was a botched 3.2.0 release. Switch to 3.2.0 instead.
```

Comment:
Should there be a header for `3.2.0` release in this changelog?

Reply:
Not yet. The axion plugin can't release 3.2.0 from the commit tagged as 3.1.5. So I need any commit on top (this PR). I'll release 3.2.0 immediately after.

But I get your point. The changelog talks about something that doesn't exist yet, so it's prone to creating chaos when my promise gets interrupted/fails. 🤔

You're right. I'll remove all mentions of the nonexistent 3.2.0